pkg/controller/controller.go

```diff
@@ -63,10 +57,7 @@ func GetCommandHandlers(opts ...Opt) ([]command.Handler, error) {
 		opt(cmdOpts)
 	}
 
-	sdsComm, err := sdscomm.New(cmdOpts.sdsServerURL, cmdOpts.agentUsername)
-	if err != nil {
-		return nil, fmt.Errorf(failCreateSDSCommErrMsg, err)
-	}
+	sdsComm := sdscomm.New(cmdOpts.sdsServerURL, cmdOpts.agentUsername)
```

Comment:
It seems that every time a WASM command gets executed, this GetCommandHandlers function gets executed as well. So in order to prevent other WASM commands from getting an error due to a blank sds server url, I moved the check into the specific command functions.